src/components/asset-amount-input/asset-amount-input.tsx

```diff
@@ -99,7 +99,7 @@ export const AssetAmountInput: FC<AssetAmountInputProps> = ({
     <>
       <View style={styles.headerContainer}>
         <Label label={label} />
-        {hasExchangeRate && (
+        {hasExchangeRate && !hideUsdSwitcher && (
```

Comment:
remove `hideUsdSwitcher` property.  Token/Usd switcher should be available if `exchangeRate` is available.  Vova did approve this